node/comparisons_test.ts

```diff
@@ -0,0 +1,110 @@
+import { assertEquals } from "../testing/asserts.ts";
+import { isDeepStrictEqual } from "./comparisons.ts";
+
+Deno.test("for basic Objects", async (t) => {
```

Comment:
These test names share the same namespace with other test cases in std library. So can you prefix the case name like `[node/comparison] for basic Objects`? (I don't think we need such prefixes for sub test steps as they are already grouped)

Reply:
Sure, I have updated these with the namespace